spec/honeykiq/server_middleware_spec.rb

```diff
@@ -23,7 +28,7 @@ def perform(arg = nil)
       'job.attempt_number': 1,
       'job.id': instance_of(String),
       'job.arguments_bytes': 2,
-      'job.latency_sec': be_within(0.01).of(0),
+      'job.latency_sec': be_within(0.05).of(0),
```

Comment:
i'm v sorry for this. if the 0.01 is serious i can probably optimise, but it was failing intermittently on my machine

Reply:
That is fine :)